packages/contracts/contracts/foundry/SphinxPluginTypes.sol

```diff
@@ -135,7 +135,13 @@ struct InitialChainState {
     bool isExecuting;
 }
 
-struct SphinxConfig {
+struct UserSphinxConfig {
+    string projectName;
+    string[] mainnets;
+    string[] testnets;
+}
```

Comment:
You might notice that there is no `orgId` field here. That is because I decided to switch to requiring the user to define an environment variable `SPHINX_ORG_ID` instead. This simplifies some things on our end since we don't have to call into the users script to fetch the org id before updating the `sphinx.lock` file. It also simplifies the UX of some of the CLI commands for the user.

For example, I've introduced a new command in this PR, `sphinx sync,` which fetches the latest `sphinx.lock` file from our backend. Since both the orgId and API key are defined as environment variables, the user doesn't have to supply arguments to this command. Likewise, we could update the `sphinx artifacts` command to only accept a project name instead of both a project name and an org id (though I have not made that change in this PR).